src/property-filter/filtering-token/index.tsx

```diff
@@ -79,7 +81,7 @@ const FilteringToken = forwardRef(
       content: editorContent,
       triggerType: 'text',
       header: editorHeader,
-      size: 'large',
+      size: popoverSize,
```

Comment:
@just-boris I added this change so that the token editor popover size depends on the enableTokenGroups flag, that is to limit the effect to the new editor variant that does require a change in size.

I noticed the effect in screenshot tests.

Reply:
ok